package.json

```diff
@@ -8,11 +8,12 @@
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
     "lint": "vue-cli-service lint",
-    "test:unit": "vue-cli-service test:unit --testPathPattern --coverage",
+    "test:unit": "vue-cli-service test:unit --testPathPattern",
```

Comment:
I _think_ the GHA CI script will still run the coverage step (but please confirm) and I'm totally OK with removing this step here.

Reply:
ah darn. I always remove this when working on the tests extensively. 
I dunno, should we remove it? Does it bother you as well, when working locally?

Reply:
It has a report upload step either way, but i'm not entirely sure if it does coverage report if we don't leave in the command. I will put this back in 👍

Reply:
Wait. Maybe we should leave this out? I personally don't like its verbosity very much when I'm running the tests locally.

Reply:
Yeah, we can have another `run` command: one with coverage and another without it :)

Reply:
I'm with ya!

Reply:
I'm OK with either approach (remove, keep, or remove but create another package,json entry).